bindings/java/c/evmc-vm.h

```diff
@@ -0,0 +1,77 @@
+/* DO NOT EDIT THIS FILE - it is machine generated */
```

Comment:
I forget to ask about this. Why is this generated file added to the repo?

Reply:
No reason. It can be removed....I just forgot to do that when I added the generation step to the makefile

Reply:
Please remove it then.